agile_problem1.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Problem1 do
+	import Integer
+	
+	#Main public function that sums up the even numbers
+	def sum_even(), do: 0

```

Comment:
Does not make sens to call even_sum with arity 0, the problem states that we want to sum of a list so there should be an arity of 1 at minimum.